test/framework/utilities.py

```diff
@@ -137,6 +137,11 @@ def setUp(self):
 
         init_config()
 
+        # disable progress bars when running the tests,
+        # since it messes up with test suite progress output when test installations are performed
```

Comment:
"since it messes with test suite progress output"
or
"since it messes up test suite progress output"

Reply:
fixed in d5ccae925